akka-actor-testkit-typed/src/main/scala/org/apache/pekko/actor/testkit/typed/CapturedLogEvent.scala

```diff
@@ -11,13 +11,14 @@ import scala.compat.java8.OptionConverters._
 import org.slf4j.Marker
 import org.slf4j.event.Level
 
-import akka.annotation.InternalApi
-import akka.util.OptionVal
+import org.apache.pekko
+import pekko.annotation.InternalApi
```

Comment:
I'd prefer to not import org.apache.pekko and to fully qualify the imports like pekko.annotation.InternalApi.

IDEs like IntelliJ issue warnings about pekko.annotation.InternalApi style imports even though they are valid.

Reply:
> I'd prefer to not import org.apache.pekko and to fully qualify the imports like pekko.annotation.InternalApi.

I did it this way because it significantly reduces the diff and it also improves the quality of the improved documentation. In some places if you use the FQN it creates so much extra boilerplace that it also changes the scalafmt

> IDEs like IntelliJ issue warnings about pekko.annotation.InternalApi style imports even though they are valid.

At least on my machine Intellij has never produced such a warning.

Reply:
I think we just keep it as FQCN, as spark and flink project is doing the same.

Reply:
Flink is irrelevant because its largely a fully Java codebase now where you have to use FQCN and Spark doesn't use paradox to generate documentation from existing Scala code. As I said before, the bigger issue is that significant portion of Akka codebase uses the `akka.<SOMETHING>` package shorthand in return types. Changing that to `org.apache.pekko.<SOMETHING>` does significantly increase the codesize (in other words its not just imports). It would also make cherry picks from Ligthbends Akka less trivial.

I am not completely against using a FQCN but I would strongly recommend changing it in a future PR, not this one and possibly only for 1.1.x. Changing to use FQCN right now would take a really long amount of time (these changes already took a long time)

Reply:
I'd say ultimately, we'd want fully-qualified imports because that's what everyone uses and also what IDEs usually create.

Regarding, the usages in code directly, I agree with your approach, it doesn't make sense to use FQCN in that case.

In any case, if you made significant changes and it works like this, cleaning up the imports can be done later, indeed. It will unfortunately lead to lots of conflicts so we should do it rather soon, though.

Reply:
If there is already consensus on this I can have a look at it, but I will do so after everything else is done.